src/Features/CSharp/Portable/MakeLocalFunctionStatic/MakeLocalFunctionStaticCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.CSharp.GetCapturedVariables;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.MakeLocalFunctionStaticWithParams
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = nameof(MakeLocalFunctionStaticWithParamsCodeRefactoringProvider)), Shared]
+    internal class MakeLocalFunctionStaticWithParamsCodeRefactoringProvider : CodeRefactoringProvider
+    {
+
+        [ImportingConstructor]
+        public MakeLocalFunctionStaticWithParamsCodeRefactoringProvider()
+        {
+        }
+
+        public override async Task ComputeRefactoringsAsync(CodeRefactoringContext context)
+        {
+
+            var (document, textSpan, cancellationToken) = context;
+
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+
+            //Gets the local function statement
+            var localFunction = await context.TryGetSelectedNodeAsync<LocalFunctionStatementSyntax>().ConfigureAwait(false);
```

Comment:
The API surface for this changed but there's a helper with very similar API here: https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/CodeRefactorings/CodeRefactoringContextExtensions.cs

Essentially just use `context.TryGetRelevantNodeAsync<LocalFunctionStatementSyntax>()` instead.